bin/tessel-2.js

```diff
@@ -99,11 +99,17 @@ makeCommand('run')
   })
   .option('slim', {
     flag: true,
-    help: 'Bundle only the required modules'
+    default: true,
+    help: 'Bundle only the required files'
   })
   .option('slimPath', {
     default: 'build.js'
   })
+  .option('full', {
```

Comment:
Should this option, and making `--slim` the default action, be added to the `push` command as well?